### ISSUE TITLE
Unable to compile v0.2.4 on Arch

### ISSUE THREAD

**Issue by Technetium1** [nix-community/nix-init#268]
I'm trying to build [nix-init](https://aur.archlinux.org/packages/nix-init) from the AUR in a clean chroot. I have made available the [nurl](https://aur.archlinux.org/packages/nurl) dependency. It fails to resolve `hyper-rustls`. It seems that since the last release, [hundreds of PRs have been merged](https://github.com/nix-community/nix-init/compare/v0.2.4...main); especially from dependabot. Maybe it is time for a new release?

Happy to provide any more information. Here is a full build log: [nix-init.log](https://builds.garudalinux.org/repos/chaotic-aur/logs/nix-init.log)

```
==> Starting build()...
    Updating crates.io index
error: failed to get `hyper-rustls` as a dependency of package `reqwest v0.11.18`
    ... which satisfies dependency `reqwest = "^0.11.18"` (locked to 0.11.18) of package `nix-init v0.2.4 (/build/nix-init/src/nix-init-0.2.4)`

Caused by:
  failed to query replaced source registry `crates-io`

Caused by:
  attempting to make an HTTP request, but --frozen was specified
==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by figsoda** [comment]
Maybe removing `--features 'reqwest/rustls-tls'` from the PKGBUILD would be able to fix the failure?

I'm planning to work on a new feature before publishing the next release, which will probably be published in the next few days.

**Comment by Technetium1** [comment]
@figsoda Great, that allowed the build to complete! What are the implications of removing it, what exactly is changed? We will wait to add this into Chaotic-AUR until the next release is published.

**Comment by figsoda** [comment]
Before 0.2.1, nix-init opted to use the `rustls-tls` feature of `reqwest` to avoid depending on openssl. nix-init 0.2.1 added dependency on the [cargo](https://docs.rs/cargo) crate which unconditionally depends on openssl, so the feature was removed in 56e2e8f4f8f29881caf0c325cad0ad5aaf26449a, and since the feature is not mentioned anywhere in `Cargo.toml`, the required dependencies (e.g. `hyper-rustls`) isn't present in `Cargo.lock` and therefore isn't fetched by the prepare phase, which causes the failure at build time.

**Comment by figsoda** [comment]
released 0.3.0

**Comment by Technetium1** [comment]
Thanks, the issue is now solved!
